README.md

```diff
@@ -44,6 +64,13 @@ Meet with your corresponding group again. And, again, select which set will be D
 
 ***Q1.4 Designers: Briefly summarize their feedback for each solution:***
 
+1. Solution 1
+  * The watch should be able to remotely connect to another interface so you don't have to download files onto the watch.
```

Comment:
Syncing a specific file sharing service or cloud storage service with a watch is certainly feasible, although I'm not sure how realistic document viewing would be on the device (as you said, limited screen space).